spec/person_spec.rb

```diff
@@ -0,0 +1,49 @@
+require_relative '../book'
+require_relative '../person'
+require_relative '../rental'
+
+describe Person do
+  person = Person.new(age: 0, name: 'name', parent_permission: true)
+  context 'Make a person' do
+    it 'Does the person have a id' do
+      expect(person.id).to be >= 0
+    end
+    it 'Does the person have the correct age' do
+      expect(person.age).to eq 0
+    end
+    it 'Does the person have the correct name' do
+      expect(person.name).to eq 'name'
+    end
+    it 'Does the person have the correct parent permission' do
+      expect(person.parent_permission).to eq true
+    end
+    it 'Default initialize values work' do
+      person_default = Person.new(age: 0)
+      expect(person_default.name).to eq 'Unknown'
+      expect(person_default.parent_permission).to eq true
+    end
+  end
+
+  context 'Test methods' do
```

Comment:
You can create a new context per method like `context '#add_rental'` or `context 'when adding rentals'`.

See the examples for more GitLab examples for further details.